conf/i18n/en/my-account.yaml

```diff
@@ -92,3 +92,28 @@ changePassword:
   changeBtn: Change password
   backBtn: Back
   feedbackMessage: Password changed successfully.
+resetPassword:
+  title: Reset Password
+  newPasswordLabelTxt: New password
+  confirmPasswordLabelTxt: Confirm new password
+  resetBtn: Reset password
+  feedbackMessage: Password reset successfully.
+forgotPassword:
+  title: Forgot Password
+  description: Please submit your email address to receive a password reset link.
+  email:
```

Comment:
Not sure how it is done now, but we should deliver the email in the language the user is in currently. I see in the German version this part is missing. Is the email always sent in English?